### ISSUE TITLE
Allow to set food modifier to 0 in the config.

### ISSUE THREAD

**Issue by pantsguyapp** [ldtteam/minecolonies-features#823]
### Prerequisites 

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [X] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version:
- MineColonies Version:
- Related Mods and their Versions:

## Is your feature request related to a problem?

I want to play without having to worry about food, but there is no such option in the config, the best I can do is set the hunger rate to really slow.

## Enhancement or Change description

Expand the food modifier limit to 0, not just 0.1.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Thodor12** [comment]
I'm not a super fan of this idea, as it would invalidate the use of several buildings (restaurant, bakery, farm).
We tend to not add things that make other things obsolete.

**Comment by pantsguyapp** [comment]
Agree, but with all things, this is not default, only if one choose to play this way, then it will be. The only buildings that I think is affected by this are the cooks, crops would still be necessary because of hospital, composter and to feed animals, which is still required for their second drops.
The feature allow for certain people to experience playing the mod in a different manner, not all pack devs will want to enable easy spoonfeed mode for the colony.
